docs/index.md

```diff
@@ -0,0 +1,12 @@
+# Welcome to the Science!
```

Comment:
This just converts from `.rst` form to `.md` form. This cannot be done for the `cli.rst` since it uses a sphinx extension that assumes `.rst`.